lmdeploy/pytorch/engine/engine.py

```diff
@@ -1016,13 +1010,13 @@ def infer(self,
         """
         token_ids = []
         for outputs in self.stream_infer(session_id,
-                                         prompt_token_ids,
+                                         input_ids,
                                          gen_config=gen_config,
                                          **kwargs):
             status, tmp_ids, _ = outputs
             if status not in [ResponseType.SUCCESS, ResponseType.FINISH]:
                 return (status, token_ids, len(token_ids))
-            token_ids += tmp_ids
+            token_ids = tmp_ids
```

Comment:
We don't need to append `tmp_id`?

Reply:
the output `token_ids` from `stream_infer` is the whole token_ids as accumulated in here
https://github.com/InternLM/lmdeploy/blob/fad7c90ffd206eddd84c2336742c4e6a66c64834/lmdeploy/pytorch/engine/engine.py#L988